uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-actions.component.js

```diff
@@ -183,8 +183,7 @@ define([
         return status.name === sharedSettings.statusNames.awaitingApproval;
       }).value;
 
-      // Check if the status before the action was awaiting approval
-      // If yes publish an event
+      // Check if the status before the action was awaiting approvalIf yes publish an event
```

Comment:
`Check if the status was "Awaiting approval" before the action. If yes, publish an event.`